GNUmakefile

```diff
@@ -149,3 +150,13 @@ format:
 	@echo "--> Running go fmt"
 	@go fmt $(PACKAGES) $(PACKAGES_IT)
 
+# -composite: avoid "literal copies lock value from fakePtr"
+.PHONY: vet
+vet:
+	@echo "--> Running go vet"
+	@go list ./... | grep -v "./vendor/*" | xargs go vet -composites
+
+.PHONY: verify-src
+verify-src: 
+	@echo "--> Checking for git changes post running tests";
```

Comment:
Why is this required? Checking source files after running tests.

Reply:
to verify if the make fmt has updated any source files. Similarly, if there is some auto gen or go deps update, we should do that as part of PR instead of letting CI silently update and build a code that doesn't match the github branch.